Update licensing-model-azure-hybrid-benefit-ahb-change.md (#9312)
MicrosoftDocs/sql-docs: azure-sql/virtual-machines/windows/licensing-model-azure-hybrid-benefit-ahb-change.md

You'll need to [register your SQL Server VM with the SQL IaaS Agent extension](sql-agent-extension-manually-register-single-vm.md).

> **Change licensing to AHB, HADR or PAYG**

Reviewer: ```suggestion
**Change licensing to AHB, HADR, or PAYG**
```

Make sure your [subscription is registered with resource provider (RP)](/azure/azure-sql/virtual-machines/windows/sql-agent-extension-manually-register-single-vm#register-subscription-with-rp).

The SQL IaaS Agent extension is required to change the license. Make sure you [remove and reinstall the SQL IaaS Agent extension](/azure/azure-sql/virtual-machines/windows/sql-agent-extension-manually-register-single-vm#register-in-full-mode) if it's in a failed state.

> **How Can I deploy SQL Server BYOL(Bring your own License) Image?**

Reviewer: ```suggestion
**How Can I deploy a SQL Server BYOL (bring your own license) image?**
```

> **How Can I deploy a SQL Server BYOL (bring your own license) image?**

Reviewer: ```suggestion
**How Can I deploy a SQL Server BYOL (bring your own license) image?**

```

> BYOL images have been retired from Azure Marketplace. You can deploy a pay-as-you-go (PAYG) gallery image and change the license to (BYOL) by enabling the [Azure Hybrid Benefit](https://azure.microsoft.com/pricing/hybrid-benefit/faq/)

Reviewer: ```suggestion
BYOL images have been retired from Azure Marketplace. You can deploy a pay-as-you-go (PAYG) gallery image and change the license to (BYOL) by enabling the [Azure Hybrid Benefit](https://azure.microsoft.com/pricing/hybrid-benefit/faq/).
```

You'll need to [register your SQL Server VM with the SQL IaaS Agent extension](sql-agent-extension-manually-register-single-vm.md).

**Change licensing to AHB, HADR, or PAYG**

Make sure your [subscription is registered with resource provider (RP)](/azure/azure-sql/virtual-machines/windows/sql-agent-extension-manually-register-single-vm#register-subscription-with-rp).

The SQL IaaS Agent extension is required to change the license. Make sure you [remove and reinstall the SQL IaaS Agent extension](/azure/azure-sql/virtual-machines/windows/sql-agent-extension-manually-register-single-vm#register-in-full-mode) if it's in a failed state.

**How Can I deploy a SQL Server BYOL (bring your own license) image?**

> BYOL images have been retired from Azure Marketplace. You can deploy a pay-as-you-go (PAYG) gallery image and change the license to (BYOL) by enabling the [Azure Hybrid Benefit](https://azure.microsoft.com/pricing/hybrid-benefit/faq/).

Reviewer 2: ```suggestion
BYOL images have been retired from Azure Marketplace. You can choose a SQL Server Standard or Enterprise edition marketplace image, and enable [Azure Hybrid Benefit](https://azure.microsoft.com/pricing/hybrid-benefit/faq/) during deployment, to create a SQL virtual machine using an existing license.
```
